### PR TITLE
fix(position): source position of CtFieldReference

### PR DESCRIPTION
**Pull request by pvojtechovsky**

### CODE REVIEW

**Comment by pvojtechovsky** [comment]
I rebased this PR. Now it is ready for review

File: src/test/java/spoon/test/position/PositionTest.java [hunk 1]
```diff
@@ -1280,4 +1283,24 @@ public void testCommentedOutClass() {
...
+		CtType<?> foo = ModelUtils.buildClass(cfg -> {
+			cfg.getEnvironment().setCommentEnabled(true);
+			cfg.getEnvironment().setAutoImports(true);
+		}, FooField.class);
+		String classContent = getClassContent(foo);
+		
+		CtAssignment<?, ?> assignment =  (CtAssignment<?, ?>) foo.getMethodsByName("m").get(0).getBody().getStatements().get(0);
+		CtFieldWrite<?> fieldWrite = (CtFieldWrite<?>) assignment.getAssigned();
+		assertEquals("FooField.f.field2", contentAtPosition(classContent, fieldWrite.getPosition()));
+//		CtFieldReference<?> fieldRef = fieldWrite.getVariable();
```
Comment (nharrand) [inline src/test/java/spoon/test/position/PositionTest.java]: Maybe remove these two commented lines if they are not used?

File: src/main/java/spoon/support/compiler/jdt/PositionBuilder.java [hunk 2]
```diff
@@ -579,11 +579,11 @@ private int getSourceEndOfTypeReference(char[] contents, TypeReference node, int
 
 	/**
 	 * @return index of first character `expectedChar`, searching forward..
-	 * Can return 'off' if it `expectedChar`.
+	 * Can return 'off' if it is `expectedChar`. return -1 if not found
```
Comment (nharrand) [inline src/main/java/spoon/support/compiler/jdt/PositionBuilder.java]:
```suggestion
	 * Can return 'off' if it is `expectedChar`. returns -1 if not found
```

**Comment by pvojtechovsky** [comment]
I changed PR as required. I am finished here.

**Comment by nharrand** [comment]
LGTM!

**Pull request merged**